pkg/config/config.go

```diff
@@ -64,6 +80,13 @@ var defaultServerConfig = &ServerConfig{
 		Port:             8089,
 		ServerReflection: true,
 	},
+	DataProxy: DataProxyConfig{
+		Upload: DataProxyUploadConfig{
+			MaxSize:               resource.MustParse("6Mi"),
+			MaxExpiresIn:          config.Duration{Duration: time.Hour},
```

Comment:
this is for fast registration, right? why so short by default?

Reply:
It's indeed for fast registration. Is an hour not enough to upload for fast register? How long do you think it should be?